src/ServicePulse.Host/app/js/views/monitored_endpoints/monitored_endpoints.controller.js

```diff
@@ -64,7 +64,7 @@
             });
 
             endpointsFromScSubscription =
-                Rx.Observable.interval(5000)
+                Rx.Observable.interval(monitoringService.calculateIntervalInMiliseconds($scope.selectedPeriod.value))
```

Comment:
Do we really want to change the rate at which we query ServiceControl?

Reply:
This address this requirement: https://github.com/Particular/LaunchWave.DevOpsAdvancedMonitoring/issues/339

Reply:
I would rather us not query SC more often. We can query the monitoring endpoint more often, but SC struggles enough as it is. And having the number refresh a bit slower (I'd say even slower than 5 seconds) is just fine for an error count @Particular/devops-launch-wave-squad

Reply:
I agree that we don't need to increase the refresh rate for data coming from SC instance.